pnpm-lock.yaml

```diff
@@ -6946,6 +6946,7 @@ packages:
 
   /@tootallnate/quickjs-emscripten@0.23.0:
     resolution: {integrity: sha512-C5Mc6rdnsaJDjO3UpGW/CQTHtCKaYlScZTly4JIu97Jxo/odCiH0ITnDXSJPTOrEKk/ycSZ0AOgTmkDtkOsvIA==}
+    requiresBuild: true
```

Comment:
this seems to repeatedly happen when I run `pnpm run clean-install` - unsure if it's bad to add 🤔